src/order/closure.lean

```diff
@@ -11,39 +11,29 @@ import tactic.monotonicity
 
 /-!
 # Closure operators between preorders
-
 We define (bundled) closure operators on a preorder as monotone (increasing), extensive
 (inflationary) and idempotent functions.
 We define closed elements for the operator as elements which are fixed by it.
-
```

Comment:
Why remove all these newlines? Blank lines are necessary in markdown to divide paragraphs.